common/arg.cpp

```diff
@@ -2436,6 +2436,13 @@ common_params_context common_params_parser_init(common_params & params, llama_ex
             }
         }
     ));
+    add_opt(common_arg(
+        {"--disable-op-offload"},
```

Comment:
For consistency with other parameters this should be renamed to `--no-op-offload`.

Reply:
Updated.